packages/playground/windows/playground-composition/HybridCRT.props

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Would this be better served saved under vnext/PropertySheets and then simply (conditionally) imported in the External/Microsoft.ReactNative.Composition.CppApp.props?